composer/datasets/utils.py

```diff
@@ -32,8 +34,8 @@ class NormalizationFn:
     Args:
```

Comment:
it has been loaded onto the device (i.e. GPU) => it has been loaded onto the device (i.e., GPU)